tests/common/test_cgroupapi.py

```diff
@@ -167,6 +163,7 @@ def controller_operation(controller):
             self.assertIn('Error in cgroup controller "cpu": A test exception.', message)
 
 
+@skip_if_predicate_true(is_systemd_present, "FileSystem cgroups API doesn't manage cgroups on systems using systemd.")
```

Comment:
why skip those tests?

Reply:
They could run and succeed on a systemd machine, but since the systemd tests don't run on a filesystem machine, I wanted to make a clear cut between the two environments and their cgroups tests.